### PR TITLE
fix: Updates sdk that fixes orderByOrgUnitName in the events repo.

### PR DESCRIPTION
**Pull request by ferdyrod**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt [hunk 1]
```diff
@@ -21,7 +21,7 @@ class ErrorModelMapper(private val fkMessage: String) {
             error.created(),
             error.httpErrorCode().toString(),
             error.errorDescription(),
-            error.errorComponent().name
+            error.errorComponent()?.name ?: ""
```
Comment (JaimeToca) [inline app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt:24]: should we show a generic message?

Comment (ferdyrod) [reply app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt:24]: Great idea, We should decide what message to show in here.

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt:24]: @Balcan Something generic ???

Comment (JaimeToca) [reply app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt:24]: @Balcan

Comment (ferdyrod) [reply app/src/main/java/org/dhis2/usescases/settings/models/ErrorModelMapper.kt:24]: @JaimeToca @Balcan Talk to @marcamsn about this. For now we will leave the space blank, so we do not need to do anything else in this PR.

**Pull request merged**